src/components/select/select.js

```diff
@@ -836,7 +836,7 @@ export class Select {
     const optionSelected = Array.from(this.select.options)
       .filter((option) => option.selected) // do not rely on getAttribute as it does not work in all cases
       .map((option) => option.text)
-      .join(', ');
+      .join('|');
```

Comment:
does it have any impact somewhere?

Reply:
this impacts here: 
![image](https://github.com/user-attachments/assets/cedfff38-2b24-4c9e-aa11-8461b4b8e0f1)